indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -46,6 +47,15 @@ def _process(self):
         return WPManageRegistration.render_template('management/regform_list.html', self.event, regforms=regforms)
 
 
+class RHManageRegistrationTags(RHManageRegFormsBase):
+    """List all registration tags for an event."""
+
+    def _process(self):
+        event_id = self.event.id
+        tags = RegistrationTag.query.filter(RegistrationTag.event_id == event_id).all()
```

Comment:
`.with_parent(self.event)` - you almost never need to use ids in queries when it comes to FKs/relationships ;)

Or even better: `tags = self.event.registration_tags.all()` (or without `.all()` if you don't make the relationship dynamic; if you don't need extra filters etc you can just use a normal lazy one)